docs/getting-started/software-management/contributing-with-git.md

```diff
@@ -0,0 +1,164 @@
+# Contributing with Git
+
+## Prerequisites
+
+### Programming
+
+Before contributing, you should have an understanding of programming and of the language a program is written in.  For example, ISIS is written in c++, and ALE is written in Python.  If you don't know how to program, you might still be able to contribute by writing documentation or finding issues.
```

Comment:
This paragraph could be more inclusive. For example, see this [open source contributing guide](https://opensource.guide/how-to-contribute/). There are a myriad of ways to contribute to a project that are not writing code. Would it be possible to write this a bit more inclusively?